module/item/systems/repair.js

```diff
@@ -140,7 +150,7 @@ class Repair {
         return await renderTemplate("systems/TheWitcherTRPG/templates/dialog/repair-dialog.hbs", templateData)
     }
 
-    async repairItem(data, simulate) {
+    async repairItem(data, simulate, gmRepair) {
```

Comment:
Due to readability I would prefer that ``simulate`` and ``gmRepair`` would be wrapped in an ``options``-object